engine/core/pom.xml

```diff
@@ -236,6 +236,11 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>17.0</version>
```

Comment:
The property ${guava.version} is already defined in the parent pom in pentaho-reporting, you shouldn't explicity put here.

Reply:
Done.